### DISCUSSION TITLE
DeltaKernal FFI error

### DISCUSSION THREAD

**Discussion by SgtGilly** [duckdb/dbt-duckdb#433] [Q&A]
Have you seen the following error before? 
```
Runtime Error in model m1 (models/m1.sql)
  IO Error: Hit DeltaKernel FFI error (from: While trying to read from delta table: 's3://...'): 
  Hit error: 8 (ObjectStoreError) with message (Error interacting with object store: Generic S3 error: 
  Error after 10 retries in 14.818550834s, max_retries:10, retry_timeout:180s, source:error sending request for url (http://169.254.169.254/latest/api/token))
```

**Comment by SgtGilly** [comment]
Getting this error when trying to use `delta_scan()` in my model.

**Comment by jwills** [comment]
No, haven't used the `delta` extension in anger yet. Some googling revealed a few different flavors of this error folks have run into, but no solutions afaict.

**Comment by owl2** [comment]
@SgtGilly 
Just to know you're trying to write on AWS S3 or in different object storage as Minio ?
